src/main/java/com/synopsys/integration/alert/common/workflow/processor/MessageContentCollector.java

```diff
@@ -148,9 +148,11 @@ protected void addItem(final List<CategoryItem> categoryItems, final CategoryIte
 
     protected final List<LinkableItem> createLinkableItemList(final LinkableItem... items) {
         final List<LinkableItem> list = new ArrayList<>();
-        if (items != null) {
+        if (null != items) {
             for (final LinkableItem item : items) {
-                list.add(item);
+                if (null != item) {
```

Comment:
Why would there be null items in this list? Are we explicitly adding something that can be null?

Reply:
The first and last names for policy are only there for policy override notifications.  So if it isn't present send a null value to avoid branching code in the policy collector.  The same is true in the edge case of losing connectivity to BlackDuck in order to get the severity of the vulnerabilities.  There I would need more complicated branching code to do the same call.  Let me look at the code another time to see if I can refactor into something clean.  I don't like checking for null either.